hugegraph-core/src/main/java/com/baidu/hugegraph/job/algorithm/SubgraphStatAlgorithm.java

```diff
@@ -23,6 +23,7 @@
 import java.util.Map;
 
 import org.apache.commons.configuration.PropertiesConfiguration;
+import org.apache.commons.configuration2.Configuration;
```

Comment:
there are still compile errors like `package javax.inject does not exist`

Reply:
see lines below